Include/code.h

```diff
@@ -119,7 +119,14 @@ PyAPI_DATA(PyTypeObject) PyCode_Type;
 #define PyCode_GetNumFree(op) (PyTuple_GET_SIZE((op)->co_freevars))
 
 /* Public interface */
+
+/* For backwards compatibility after adding posonlyargcount */
```

Comment:
IMHO this comment can be removed. There is no **need** to deprecate anything so far.